packages/react-components/react-icons-compat/library/src/icon.ts

```diff
@@ -209,19 +208,20 @@ function _warnDuplicateIcon(iconName: string): void {
   if (!options.disableWarnings) {
     _missingIcons.push(iconName);
     if (_missingIconsTimer === undefined) {
-      // eslint-disable-next-line no-restricted-globals
-      _missingIconsTimer = setTimeout(() => {
-        // eslint-disable-next-line no-console
-        console.warn(
-          `Some icons were re-registered. Applications should only call registerIcons for any given ` +
-            `icon once. Redefining what an icon is may have unintended consequences. Duplicates ` +
-            `include: \n` +
-            _missingIcons.slice(0, maxIconsInMessage).join(', ') +
-            (_missingIcons.length > maxIconsInMessage ? ` (+ ${_missingIcons.length - maxIconsInMessage} more)` : ''),
-        );
-        _missingIconsTimer = undefined;
-        _missingIcons = [];
-      }, warningDelay);
+      _missingIconsTimer =
+        // eslint-disable-next-line @nx/workspace-no-restricted-globals
```

Comment:
same as before, turning off should be right next to problematic code reference => more extensive code re-structuring